src/uct/ib/dc/dc_mlx5.c

```diff
@@ -2050,10 +2040,9 @@ static UCS_CLASS_CLEANUP_FUNC(uct_dc_mlx5_iface_t)
     uct_dc_mlx5_ep_t *ep, *tmp;
 
     ucs_trace_func("");
-    uct_base_iface_progress_disable(&self->super.super.super.super,
+    uct_base_iface_progress_disable(&self->super.super.super.super.super,
                                     UCT_PROGRESS_SEND | UCT_PROGRESS_RECV);
     uct_dc_mlx5_iface_cleanup_dcis(self);
-    uct_rc_mlx5_iface_common_cleanup(&self->mlx5_common);
     uct_dc_mlx5_iface_tag_cleanup(self);
 
     uct_dc_mlx5_destroy_dct(self);
```

Comment:
also no need to destroy dct twice now. Destroying dct can removed from uct_dc_mlx5_iface_tag_cleanup